subjects/devops/play-with-containers-py/audit/README.md

```diff
@@ -0,0 +1,225 @@
+#### General
+
+##### Check the Repo content
+
+A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.
+
+###### Are all the required files present?
+
+###### Does the `.env` file not exist in the repo?
+
+###### Are all pushed files in the repo clean of any credentials or passwords?
+
+##### Ask the following questions to the group or student:
+
+##### What are containers and what are their advantages?
+
+##### What is the difference between containers and virtual machines?
+
+##### What is Docker and what is it used for?
+
+###### Did the student reply correctly to the questions?
+
+##### Open and read the `README.md` file provided by the student.
+
+###### Does the `README.md` file contain all the required information to run and manage the solution (prerequisites, configuration, setup, usage, etc)?
+
+#### Check the student infrastructure:
+
+The student must implement this architecture:
+![architecture](../pictures/architecture.png)
+
+##### Run the student infrastructure:
+
+```console
+user:~$ docker-compose up
+<...>
+inventory-database   ... done
+billing-database     ... done
+inventory-app        ... done
+billing-app          ... done
+RabbitMQ             ... done
+api-gateway-app      ... done
+<...>
+user:~$
+```
+
+###### Does the student architecture reflect the infrastructure enforced by the subject?
+
+###### Does the infrastructure start correctly?
+
+##### Ask the following questions to the group or student
+
+##### What is a microservices' architecture?
```

Comment:
extra apostrophe after "microservices."

Reply:
It is my grammar autocorrect that suggested adding the apostrophe :)